app/controllers/idv/sessions_controller.rb

```diff
@@ -31,6 +36,19 @@ def update
 
     private
 
+    def validation_errors
+      @_validation_errors ||= []
```

Comment:
I would recommend using a Form Object and Validator for this logic as opposed to putting it in the controller. See our `/forms` and `/validators` directories for examples.